src/connector/src/sink/iceberg.rs

```diff
@@ -849,7 +849,10 @@ impl SinkCommitCoordinator for IcebergSinkCommitter {
             .iter()
             .map(|meta| WriteResult::try_from(meta, &self.partition_type))
             .collect::<Result<Vec<WriteResult>>>()?;
-
+        if write_results.is_empty() || write_results.iter().all(|r| r.data_files.is_empty()) {
+            tracing::info!("No data to commit in epoch {epoch}.");
```

Comment:
```suggestion
            tracing::debug!(?epoch, "no data to commit");
```

Otherwise seems like it would flood our flogs.

Reply:
I think it's a rare case